### PR TITLE
Dts i2c client cleanup

### PR DESCRIPTION
**Pull request by galak**

### CODE REVIEW

File: drivers/gpio/gpio_sx1509b.c [hunk 1]
```diff
@@ -14,12 +14,6 @@
 #include <sys/byteorder.h>
```
Comment (pabigot) [inline drivers/gpio/gpio_sx1509b.c]: Please drop the changes in this file; the equivalent has been done in the updated version in topic-gpio, and doing it  here now will just create conflicts.

**Review by pabigot — CHANGES_REQUESTED** [review]

**Review by Mani-Sadhasivam — APPROVED** [review]
LED part looks good!

**Comment by Mani-Sadhasivam** [comment]
@galak Could you please fix the checkpatch warnings also?

File: drivers/gpio/gpio_sx1509b.c (hunk 1 above)
Comment (galak) [reply drivers/gpio/gpio_sx1509b.c]: I'm fine w/dropping it here, the other option is when I rebase topic-gpio we'll pick this up.  Thoughts? [i'll handle the merge conflict]

**Comment by galak** [comment]
> @galak Could you please fix the checkpatch warnings also?

should be fixed now.

File: drivers/gpio/gpio_sx1509b.c (hunk 1 above)
Comment (pabigot) [reply drivers/gpio/gpio_sx1509b.c]: Either way, I guess. G'head and keep it here.

**Review by pabigot — APPROVED** [review]

**Pull request merged**